CMakeLists.txt

```diff
@@ -10,7 +10,27 @@ if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
     set(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT False)
 endif()
 
+# Set default build type to Release
+if(NOT CMAKE_BUILD_TYPE)
+  messageInfo("No CMAKE_BUILD_TYPE, using default CMAKE_BUILD_TYPE=Release")
+  set(CMAKE_BUILD_TYPE "Release")
+endif()
+
```

Comment:
Put this before the test `if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)` and use `CMAKE_BUILD_TYPE` as suffix for installed directory (as for Radium-bundle). 
Keep the default install as `${CMAKE_CURRENT_BINARY_DIR}/installed-${CMAKE_CXX_COMPILER_ID}[-${CMAKE_BUILD_TYPE }]` with no suffix to for release install.